src/dirtyfields/compat.py

```diff
@@ -13,6 +14,11 @@ def is_db_expression(value):
         return isinstance(value, (BaseExpression, Combinable))
 
 
+def is_deferred(model, field):
```

Comment:
Actually I wonder if `compat.py` is not a good option. This method works with Django 1.4 to 1.8, but as it plays with Django's internals nothing tells us that is will still be the case in the future.
Putting it in `compat.py` is a way of abstracting any API change.